atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/CommitLockTest.java

```diff
@@ -65,16 +66,18 @@ public class CommitLockTest extends TransactionTestSetup {
     private static final String COLUMN = "col_1";
     private static final String OTHER_COLUMN = "col_2";
 
-    @DataPoints
-    public static ConflictHandler[] conflictHandlers = ConflictHandler.values();
+    private static List<ConflictHandler> conflictHandlers() {
+        return Arrays.asList(ConflictHandler.values());
+    }
 
     public CommitLockTest() {
         super(TRM, TRM);
     }
 
-    @Theory
```

Comment:
As far as I understand, theories work in the following way:
- We obtain the Cartesian product of data points and feed them to each test.

Since we have only one data point, we didn't use the theory in the first place. Anyway, it is not supported in JUnit5, so just use the parameterized test.